weaver/wps_restapi/providers/providers.py

```diff
@@ -161,9 +131,79 @@ def remove_provider(request):
 @sd.provider_service.get(tags=[sd.TAG_PROVIDERS], renderer=OUTPUT_FORMAT_JSON,
                          schema=sd.ProviderEndpoint(), response_schemas=sd.get_provider_responses)
 @log_unhandled_exceptions(logger=LOGGER, message=sd.InternalServerErrorResponseSchema.description)
+@check_provider_requirements
 def get_provider(request):
     """
     Get a provider definition (GetCapabilities).
     """
     service, _ = get_service(request)
     return HTTPOk(json=service.summary(request))
+
+
+@sd.provider_processes_service.get(tags=[sd.TAG_PROVIDERS, sd.TAG_PROCESSES, sd.TAG_PROVIDERS, sd.TAG_GETCAPABILITIES],
+                                   renderer=OUTPUT_FORMAT_JSON, schema=sd.ProviderEndpoint(),
+                                   response_schemas=sd.get_provider_processes_responses)
+@log_unhandled_exceptions(logger=LOGGER, message=sd.InternalServerErrorResponseSchema.description)
+@check_provider_requirements
+def get_provider_processes(request):
```

Comment:
Does this add new provider's endpoints? I saw a refactor point in the change log, but it looks like we have more providers functionnalities added here.

Reply:
No new endpoints, I only moved them out of `processes` and into the correct `providers` file.
They were placed there to resolve some circular imports at some point, but a previous refactoring when bumping to 4.0 made it so that this is not needed anymore.